packages/bot-web-ui/src/components/dashboard/dashboard-component/index.scss

```diff
@@ -146,13 +218,65 @@
     }
 }
 
-.db-tour {
-    text-align: left;
+.disabled--card {
+    pointer-events: none;
+    cursor: not-allowed;
+    opacity: 0.6;
+}
+
+.load-strategy__button-group {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin-right: 1.6rem;
+}
+
+.load-strategy__button-group--clear {
+    height: 3.4rem;
+    width: 6.7rem;
+    background-color: #d6d6d6;
+    border-radius: 0.4rem;
+    color: #ffffff;
+    outline: none;
+    border: none;
+    font-weight: 700;
+    font-size: 1.4rem;
+    cursor: pointer;
+    margin-right: 1.6rem;
+}
+
+.load-strategy__button-group--open {
+    height: 3.4rem;
+    width: 6.7rem;
+    background-color: #ff444f;
+    border-radius: 0.4rem;
+    color: #ffffff;
+    outline: none;
+    border: none;
+    font-weight: 700;
+    font-size: '1.4rem';
+    cursor: pointer;
+}
+
+.dc-dialog__wrapper--googledrive {
```

Comment:
Why are we writing this as a class? are we overriding a class? can we separate all the related css files to a their folders?

Reply:
done ✅